src/components/common/RootAssetSelect/RootAssetSelect.test.tsx

```diff
@@ -28,21 +32,29 @@ describe('RootAssetSelect', () => {
   });
 
   it('onSelectAsset should be triggered', async done => {
```

Comment:
@mrtraser Can you please check this test. Looks like there is some issue to emulate onChange event after finding the Select element. I've tried on both wrapped and pure component exports, different instances, spy functions but onSelectAsset is not changing component's state.